controllers/members.js

```diff
@@ -83,11 +84,22 @@ const archiveMembers = async (req, res) => {
   try {
     const { username } = req.params;
     const user = await dataAccess.retrieveUsers({ username });
+    const superUserId = req.userData.id;
+    const body = req.body;
+    const meta = {
+      userId: user?.user?.id,
+      superUserId: superUserId,
+      username: username,
+    };
+    if (!body?.reason || /^\s*$/.test(body?.reason)) {
+      return res.boom.badRequest("Reason is required");
```

Comment:
`|| /^\s*$/.test(body?.reason)`--> why??

Reply:
Just in case the reason a super user will mention should not be `empty space`, it should have proper text

Reply:
`!body?.reason ` itself not handle empty space condition??

Reply:
no, because empty space has length 1, so it will not validate properly
Also, the current implementation can handle multiple empty spaces to make validation properly